README.md

```diff
@@ -7,7 +7,7 @@ compatible hook for [Google Stackdriver logging](https://cloud.google.com/loggin
 
 Install in the usual Go way:
 ```sh
-go get -u github.com/knq/sdhook
+go get -u github.com/kenshaw/sdhook
```

Comment:
Well, since we forked it, shouldn't it say
```suggestion
go get -u github.com/thermeon/sdhook
```

Reply:
Thank you for your comment. Done